frontend/src/app/globals.css

```
@@ -5,42 +5,44 @@
+  --chart-3: #a3f5d1;
+  --chart-4: #a3d1f5;
+  --chart-5: #d1a3f5;
+  --sidebar: #f3f3f4;
+  --sidebar-foreground: #666770;
+  --sidebar-primary: #dddde0;
+  --sidebar-primary-foreground: #28282d;
+  --sidebar-accent: #e7e7e9;
+  --sidebar-accent-foreground: #28282d;
+  --sidebar-border: #d9d9dc;
+  --sidebar-ring: #00b0f4;
+  --font-sans: Noto Sans JP, ui-sans-serif, sans-serif, system-ui;
```

Stylelint (17.9.0)

[error] Expected "Noto" to be "noto" (value-keyword-case)

(value-keyword-case)